### ISSUE TITLE
AH allowing more than 7 listings

### ISSUE THREAD

**Issue by TeoTwawki** [DarkstarProject/darkstar#2673]
Logged into whasf's oldschool server to verify. I was able to list far more than 7 slots worth.

Please note that if you at all check your sales status, the client will impose its own restriction for the duration of the session. If you never check sales history, client relies on the server to restrict it. The server restriction is somehow broke but used to work.

**Comment by Fiocitrine** [comment]
Hmm, this has been an issue since I've been on Old School. I thought this issue was known. Also, like many others on our server, I have a lot of stuff on AH. Will the fix, provided it's fixed, erase all those other items on the AH past the last/first seven?

**Comment by TeoTwawki** [comment]
no, would prevent additional listings from the player till their list sold down to less than 7.

Used to (some servers still do) apply a small hack to allow more than seven (should probably be a setting in the confs really).

**Comment by teschnei** [comment]
im not going to make it configurable if the client enforces it

**Comment by TeoTwawki** [comment]
> probably

just sayin, its a common thing ppl fiddle with

**Comment by Scavenge** [comment]
even though kj sounds uninterested, if you're going to suggest making that a conf setting then you will want to maybe consider an option to choose how many listings to show on sales status, because we had to make a php tool on our website to be able to take down listings that were not visible in sales status (and that involves just changing the price so you can buy it back, kinda risky)

**Comment by TeoTwawki** [comment]
@Scavenge not possible as thats done in client which is one of the reasons we won't be getting a conf setting.

**Comment by TeoTwawki** [comment]
I had also forgotten that hundreds of an item listed can break that item on the AH...Did you folks at Era ever fix those blood stones? The were stuck for over a year from one person listing so many.

**Comment by R3P0FFXI** [comment]
@teschnei any news on this bug, this still seems to be an issue since @TeoTwawki reported this on latest master. If you do not check AH history you can bypass the client hardlock of 7 items and get unlimited AH slots to sell items.
